languages/elixir/exercises/concept/keyword-lists/.docs/introduction.md

```diff
@@ -0,0 +1,16 @@
+Keyword lists are a key-value data structure.
+
+```elixir
+[month: "April", year: 2018]
+```
+
+Keyword lists are lists of `{key, value}` tuples, and can also be written as such, but the shorter syntax is more widely used.
+
+```elixir
+[month: "April"] == [{:month, "April"}]
+# => true
+```
+
+Keys in a keyword list must be atoms, but the values can be anything. Each key can be used more than once. The key/value pairs in a keyword list are ordered.
+
+You can work with keyword lists using the same methods as for lists, or you can use the `Keyword` module.
```

Comment:
```suggestion
You can work with keyword lists using the same functions as for lists, or you can use the `Keyword` module.
```

I think you may also need to explain how key value pairs that trail the end of a function call are not variadic like in  javascript or python, but instead are a shortcut for explicitly declaring a keyword list for the third argument.

```elixir
# Explain that this
call_function(arg, option: 1, option: 2)

# Is equivalent
call_function(arg, [option: 1, option: 2])

Reply:
In this sentence, I meant the word "method" in its non-programming meaning. Would "approaches" be better?

Having looked up what "variadic" meant, I agree I should mention that. But I think it fits better in `after.md` than here.

Reply:
I think approaches would work, I am just trying in my own writing to just be cautious of OOP terms.

Is there a better plain-language term for variadic? I have always wondered since I don't think I can even say variadic.

Reply:
I don't think there are any synonyms, I didn't even know the word, so I would always just describe it as "not having a fixed arity".

Reply:
"approaches" is good to me.